core/tests/protractor/userJourneys.js

```diff
@@ -268,11 +271,66 @@ describe('Oppia static pages tour', function() {
 describe('Site language', function() {
   var adminPage = null;
   var libraryPage = null;
+  var creatorDashboardPage = null;
+  var collectionEditorPage = null;
+  var firstExplorationId = null;
+  var collectionId = null;
 
-  beforeEach(function() {
+  beforeAll(function() {
     adminPage = new AdminPage.AdminPage();
     libraryPage = new LibraryPage.LibraryPage();
+    creatorDashboardPage = new CreatorDashboardPage.CreatorDashboardPage();
+    collectionEditorPage = new CollectionEditorPage.CollectionEditorPage();
 
+    var EDITOR_USERNAME = 'langCollections';
+    var CREATOR_USERNAME = 'langCreatorExplorations';
+
+    users.createUser('lang@collections.com', EDITOR_USERNAME);
```

Comment:
Why have this run before all tests if you only use this setup in one of them?

Reply:
Done!